canonical/cloud-init
# .travis.yml: don't run cloud_tests in CI

## Proposed Commit Message
```
.travis.yml: don't run cloud_tests in CI

We have replicated their functionality in the new integration testing
framework, and have seen that running for long enough to be confident in
relying on it.

cloud_tests are still used in cloud-init's nightly test runs: this is
only modifying what we run in Travis.
```

## Additional Context

Opening this as draft to make sure we have consensus on doing this before it lands.

## Test Steps

Observe that only one integration testing job is scheduled in the Travis run for this PR.

## Checklist:
